ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
+nodejs_app_git_repo: https://github.com/gpii/universal.git
+nodejs_app_git_branch: master
+nodejs_app_npm_packages:
+  - grunt-cli
```

Comment:
This comes by default with the nodejs role -- see https://github.com/idi-ops/ansible-nodejs/blob/master/vars/RedHat.yml#L30-L31